Unity-Technologies/ScriptableRenderPipeline
# Move Universal MockHMD tests into new separate project

### Purpose of this PR
Moving MockHMD tests out of UniversalGraphicsTest project into new Universal_Stereo project per Graphics Team's request. 

---
### Release Notes
Disabled Universal_Stereo tests on Mac due to limited market adoption of VR on Mac, and screen capture on Mac is monoscopic due to a ug that is unlikely to ever be fixed.  

---
### Testing status
**Katana Tests**: 
My dev branch runs in parity with master on Yamato for ScriptableRenderPipeline "run all test projects on all platforms" builder.  My dev branch does not appear in the drop-down menu in Katana for the com.unity.render-pipelines builders.  Per discussion with Sophia C. she is investigating whether additional set-up is needed to make the Universal_Stereo tests appear in these builders. 

**Manual Tests**: What did you do?
- [YES] Opened test project + Run graphic tests locally
- [NO; known failures in master for Windows Standalone tests block mine. ] Built a player
- [N/A; not introducing new UI. ] Checked new UI names with UX convention
- [N/A; PR only adds and removes tests. ] Tested UI multi-edition + Undo/Redo
- [YES; ImportFBX and Blit.shader error are the same as master.] C# and shader warnings (supress shader cache to see them)
- [N/A; not adding Resources in this PR.] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

I tried to run on Katana by typing my branch name into the link in place of master but got the following error in stdio: 
[19:51:36] fatal: Couldn't find remote ref usrp_mockhmd_dennisd 
[19:51:36] fatal: the remote end hung up unexpectedly\n

Here is the URL I used: 
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders/proj57-All%20Tests/builds/4191?automation-tools_branch=usrp_mockhmd_dennisd&ScriptableRenderLoop_branch=master&unity_branch=trunk

Yamato is green for the "run all test projects on all platforms" builder in ScriptableRenderPipeline except "Run OSX Metal playmode tests on katana" which is okay per Sophia. the URL is: 

https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/usrp_mockhmd_dennisd/.yamato%2Fupm-ci-projects.yml%23all_project_ci/305977/job

Any test projects to go with this to help reviewers?
My whole PR is a test project.

---
### Overall Product Risks
**Technical Risk**: Low.

**Halo Effect**: None.

---
### Comments to reviewers
I am a Yamato newbie, so let me know if you see anything wrong with my YAML files. 
Also keep me in the loop if action is needed from me to ensure universal_Stereo runs on Katana.
